src/main/java/org/folio/circulation/domain/representations/Error.java

```diff
@@ -0,0 +1,31 @@
+
+package org.folio.circulation.domain.representations;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.folio.circulation.domain.representations.anonymization.Parameter;
+
+public class Error {
```

Comment:
Was there a reason the existing `ValidationError` couldn't be used, or was that not known about?